code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -112,3 +112,24 @@
 /obj/item/projectile/bullet/seed
 	damage = 4
 	stamina = 1
+
+//////////////////
+//    20 BORE   //
+//////////////////
+
+/obj/item/projectile/bullet/pellet/shotgun_buckshot/twentybore_buckshot
+	name = "20g buckshot pellet"
+	damage = 10
+
+/obj/item/projectile/bullet/shotgun_slug/twentybore_slug
+	name = "12g shotgun slug"
+	damage = 50
+
+/obj/item/projectile/bullet/shotgun_beanbag/twentybore_beanbag
+	name = "beanbag slug"
+	damage = 2
+	stamina = 50
+
+/obj/item/projectile/bullet/incendiary/shotgun/twentybore_incendiary
+	name = "incendiary slug"
+	damage = 16
```

Comment:
Just as a slight heads up from experience. Anyone using incendiaries is aiming to set people on fire, not necessarily looking for direct bullet damage. Both this and the 12g will hit for the same fire stacks (4). Which makes them pretty much indistinguishable for the function you want from them.

Reply:
I kinda just wanted to lower the bullet damage because it's a smaller shell. Sadly I ain't American so I have no idea if the hot flash from however much powder is in the 12G round is going to set people on fire more than however much powder is in the 20G round, sadly I can't test it and find out.
It is admittedly more or less the same as the 12G incendiary one which does 20 damage (I think) and the 4 firestacks, I could lower the direct damage by a lot and make it rely more or less solely on the firestacks to deliver damage? I guess the real damage that comes from this isn't as much the burn damage from being set on fire, I guess it's vulnerability you have to put yourself in to put it out, unless you can find an extinguisher.

Reply:
Don't worry about realism for the same of accuracy to anything in reality. Think about this from a gameplay perspective. If you want to differentiate the two, it's not a bad move to weaken the bullets firestacks.

Reply:
Marking as resolved since it's being removed.
(Seriously though, thanks for all your time going over this.)